test/e2e/alertmanager_test.go

```diff
@@ -28,86 +28,87 @@ import (
 )
 
 func TestAlertmanagerCreateDeleteCluster(t *testing.T) {
+	t.Parallel()
+
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup(t)
+	ns := ctx.CreateNamespaceInCtx(t, framework.KubeClient)
```

Comment:
The `InCtx` suffix is unnecessary here I think, as the `CreateNamespace` method would be bound to the ctx struct anyways, that's "context" enough I think.